update data jobs monitoring setup guide wrt databricks integration permissions (#23676)
DataDog/documentation: content/en/data_jobs/databricks.md

### Configure the Datadog-Databricks integration

> 1. In your Databricks workspace, go to **Settings** > **Developer**. Next to **Access tokens**, click **Manage**.

Reviewer: ```suggestion
1. In your Databricks workspace, click on your profile in the top right corner and go to **Settings**. Select **Developer** in the left side bar. Next to **Access tokens**, click **Manage**.
```

> 1. Click **Generate new token**, enter a comment, leave the Lifetime as empty and click **Generate**. Take note of your token.

Reviewer 2: ```suggestion
1. Click **Generate new token**, enter a comment, leave the Lifetime as empty, and click **Generate**. Take note of your token.
```

> 1. As an alternative, follow [the guide][10] to generate access token for a [service principal][11].

Reviewer 2: ```suggestion
1. As an alternative, follow the [official Databricks documentation][10] to generate access token for a [service principal][11].
```

> 1. Click **Generate new token**, enter "Datadog Integration" in the **Comment** field, remove the default value in **Lifetime (days)**, and click **Generate**. Take note of your token. **Important:** Make sure you delete the default value in **Lifetime (days)** so that the token doesn't expire and the integration doesn't break.

Reviewer: Is it possible to get the **Important:...**  callout to be on a newline but not create a separate step in markdown?

> 1. Ensure the account generating the token has [CAN VIEW][9] access for Databricks Jobs you want to monitor and the token is configured to never expire.

Reviewer 2: ```suggestion
1. Ensure the account generating the token has [CAN VIEW access][9] for the Databricks jobs you want to monitor, and that the token is configured to never expire.
```

> 1. Ensure the account generating the token has [CAN VIEW access][9] for the Databricks jobs you want to monitor, and that the token is configured to never expire.

Reviewer: ```suggestion
1. Ensure the account generating the token has [CAN VIEW access][9] for the Databricks jobs and clusters you want to monitor, and that the token is configured to never expire.
```

Reviewer: Also, the "as an alternative..." and "ensure account generating token ...." shouldn't be explicity steps, but additional details/alternatives on step 2
